edb/edgeql/compiler/viewgen.py

```diff
@@ -1097,9 +1097,8 @@ def _normalize_view_ptr_expr(
         )
 
     # Prohibit invalid operations on id and __type__
-    ptrcls_sn = ptrcls.get_shortname(ctx.env.schema)
     id_access = (
-        ptrcls_sn.name == 'id'
```

Comment:
I have a nagging suspicion that this might not be the only such place.

Reply:
Yeah.

Maybe we should just ban it?